mmdet3d/models/detectors/base.py

```diff
@@ -104,4 +104,4 @@ def show_results(self, data, result, out_dir):
                 ValueError(
                     f'Unsupported box_mode_3d {box_mode_3d} for convertion!')
             pred_bboxes = pred_bboxes.tensor.cpu().numpy()
-            show_result(points, None, pred_bboxes, out_dir, file_name)
+            show_result(points, None, pred_bboxes, out_dir, file_name, False)
```

Comment:
Is it necessary to change the default behavior of `show_results`.

Reply:
It is unnecessary, I will modified in a new PR